src/library/entities/Library.java

```diff
@@ -84,153 +86,155 @@ public static synchronized void SaVe() {
+        Loan loan = new Loan(getNextLoanId(), item, patron, dueDate);
+        patron.TaKe_OuT_LoAn(loan);
+        item.TaKeOuT();
+        loans.put(loan.GeT_Id(), loan);
+        currentLoans.put(item.GeTiD(), loan);
         return loan;
     }
     
     
-    public Loan GeT_LoAn_By_ItEm_Id(long ITem_ID) {
-        if (CuRrEnT_LoAnS.containsKey(ITem_ID)) 
-            return CuRrEnT_LoAnS.get(ITem_ID);
-        
+    public Loan getLoanByItemId(long itemId) {
+        if (currentLoans.containsKey(itemId)) {
+            return currentLoans.get(itemId);
+        }
         return null;
     }
 
     
-    public double CaLcUlAtE_OvEr_DuE_FiNe(Loan LoAn) {
-        if (LoAn.Is_OvEr_DuE()) {
-            long DaYs_OvEr_DuE = Calendar.GeTiNsTaNcE().GeTDaYsDiFfErEnCe(LoAn.GeT_DuE_DaTe());
-            double fInE = DaYs_OvEr_DuE * FiNe_PeR_DaY;
-            return fInE;
+    public double calculateOverDueFine(Loan loan) {
+        if (loan.Is_OvEr_DuE()) {
+            long daysOverDue = Calendar.GeTiNsTaNcE().GeTDaYsDiFfErEnCe(loan.GeT_DuE_DaTe());
+            double fine = daysOverDue * FINE_PER_DAY;
+            return fine;
         }
         return 0.0;        
     }
 
 
-    public void DiScHaRgE_LoAn(Loan cUrReNt_LoAn, boolean iS_dAmAgEd) {
-        Patron PAtrON = cUrReNt_LoAn.GeT_PaTRon();
-        Item itEM  = cUrReNt_LoAn.GeT_ITem();
+    public void dischargeLoan(Loan currentLoan, boolean isDamaged) {
+        Patron patron = currentLoan.GeT_PaTRon();
+        Item item  = currentLoan.GeT_ITem();
         
-        double oVeR_DuE_FiNe = CaLcUlAtE_OvEr_DuE_FiNe(cUrReNt_LoAn);
-        PAtrON.AdD_FiNe(oVeR_DuE_FiNe);    
+        double overDueFine = calculateOverDueFine(currentLoan);
+        patron.AdD_FiNe(overDueFine);
         
-        PAtrON.dIsChArGeLoAn(cUrReNt_LoAn);
-        itEM.TaKeBaCk(iS_dAmAgEd);
-        if (iS_dAmAgEd) {
-            PAtrON.AdD_FiNe(damageFee);
-            DaMaGeD_ItEmS.put(itEM.GeTiD(), itEM);
+        patron.dIsChArGeLoAn(currentLoan);
+        item.TaKeBaCk(isDamaged);
+        if (isDamaged) {
+            patron.AdD_FiNe(DAMAGE_FEE);
+            damagedItems.put(item.GeTiD(), item);
         }
-        cUrReNt_LoAn.DiScHaRgE();
-        CuRrEnT_LoAnS.remove(itEM.GeTiD());
+        currentLoan.DiScHaRgE();
+        currentLoans.remove(item.GeTiD());
     }
 
 
-    public void UpDaTe_CuRrEnT_LoAnS_StAtUs() {
-        for (Loan lOaN : CuRrEnT_LoAnS.values()) 
-            lOaN.UpDaTeStAtUs();
-                
+    public void updateCurrentLoansStatus() {
+        for (Loan loan : currentLoans.values()) {
+            loan.UpDaTeStAtUs();
+        }
     }
 
 
-    public void RePaIrITem(Item cUrReNt_ItEm) {
-        if (DaMaGeD_ItEmS.containsKey(cUrReNt_ItEm.GeTiD())) {
-            cUrReNt_ItEm.rEpAiR();
-            DaMaGeD_ItEmS.remove(cUrReNt_ItEm.GeTiD());
+    public void repairItem(Item currentItem) {
+        if (damagedItems.containsKey(currentItem.GeTiD())) {

```

Comment:
See https://github.com/elliotmitchum/ITC205/pull/35/files#r934038851

Reply:
commit -m"refining repairItem method to conform with CSG10"
commit e01c4afba8a694b9fe5f9c2ffda1bc70cbc326e7 
@elliotmitchum  @haynescrossley